server/pulp/server/async/scheduler.py

```diff
@@ -12,13 +12,13 @@
 from pulp.common import constants
 from pulp.common.dateutils import ensure_tz
 from pulp.server.async import worker_watcher
-from pulp.server.async.celery_instance import celery as app
-from pulp.server.async.tasks import _delete_worker
 from pulp.server.db import connection as db_connection
 from pulp.server.db.connection import UnsafeRetry
 from pulp.server.db.model.dispatch import ScheduledCall, ScheduleEntry
 from pulp.server.db.model import Worker, CeleryBeatLock
 from pulp.server.managers.schedule import utils
+from pulp.tasking.celery_instance import celery as app
+from pulp.tasking.system import _delete_worker
```

Comment:
Since this is used outside the module it seems more appropriate to remove the `_` prefix.